### PR TITLE
Edit rss feed 

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
Edit one rss feed to include only technical posts.
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ module.exports = {
               {
                 allPosts: allMdx(
                   sort: { fields: [frontmatter___date], order: DESC }
-                  filter: { fields: { collection: { in: ["blog", "news"] } }, frontmatter: { published: { eq: true }, featured: { eq: true }, category: { eq: "Announcements" } } }
+                  filter: { fields: { collection: { in: ["blog", "news"] } }, frontmatter: { published: { eq: true }, category: { nin: ["Programs", "Community", "Events"] } } }
```
Comment (debo19) [inline gatsby-config.js:84]:
```suggestion
                  filter: { fields: { collection: { in: ["blog", "resources", "news"] } }, frontmatter: { published: { eq: true }, category: { nin: ["Programs", "Community", "Events"] } } }
```

**Comment by debo19** [comment]
@leecalcote  Yeah, I've included news too

**Comment by leecalcote** [comment]
Thanks for confirming. The `nin` logic is potentially dangerous. We will need to be very aware that all other posts will be promoted on reddit and hacker news, otherwise, the account runs the risk of being marked as spam.

**Comment by leecalcote** [comment]
@debo19 a private date field for resources for RSS, not visible sorting (since that categorization is all topical) - yes, makes sense.

**Pull request merged**